website/docs/r/cse_kubernetes_cluster.html.markdown

```diff
@@ -0,0 +1,363 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_cse_kubernetes_cluster"
+sidebar_current: "docs-vcd-resource-cse-kubernetes-cluster"
+description: |-
+  Provides a resource to manage Kubernetes clusters in VMware Cloud Director with Container Service Extension installed and running.
+---
+
+# vcd\_cse\_kubernetes\_cluster
+
+Provides a resource to manage Kubernetes clusters in VMware Cloud Director with Container Service Extension (CSE) installed and running.
+
+Supported in provider *v3.12+*
```

Comment:
Should we briefly declare VCD version requirements (at least the minimum) or is it too hard to maintain?

Reply:
The good thing is that the VCD version is more of a pre-requisite to have CSE installed, so if someone managed to do it, it would mean they can use the resource.

For that reason, the VCD version is stated in the [install guide (pre-requisites section)](https://github.com/vmware/terraform-provider-vcd/blob/main/website/docs/guides/container_service_extension_4_x_install.html.markdown#pre-requisites) rather than here.

Anyway, if it can avoid misunderstandings I could add a note, let's say "To check compatibility with VCD, refer to the install guide"

Reply:
> The good thing is that the VCD version is more of a pre-requisite to have CSE installed, so if someone managed to do it, it would mean they can use the resource.

I am good :)